fuzzers/nyx_libxml2_parallel/src/main.rs

```diff
@@ -0,0 +1,103 @@
+use std::path::{Path, PathBuf};
+
+use libafl::bolts::launcher::Launcher;
+use libafl::bolts::rands::StdRand;
+use libafl::bolts::shmem::ShMemProvider;
+use libafl::events::EventConfig;
+use libafl::Error;
+use libafl::{
+    bolts::{core_affinity::Cores, rands::RandomSeed, shmem::StdShMemProvider, tuples::tuple_list},
+    corpus::{Corpus, InMemoryCorpus, OnDiskCorpus, Testcase},
+    feedbacks::{CrashFeedback, MaxMapFeedback},
+    inputs::BytesInput,
+    monitors::MultiMonitor,
+    mutators::{havoc_mutations, StdScheduledMutator},
+    observers::StdMapObserver,
+    schedulers::RandScheduler,
+    stages::StdMutationalStage,
+    state::StdState,
+    Fuzzer, StdFuzzer,
+};
+use libafl_nyx::executor::NyxExecutor;
+use libafl_nyx::helper::NyxHelper;
+
+fn main() {
+    let shmem_provider = StdShMemProvider::new().expect("Failed to init shared memory");
+    let broker_port = 7777;
+
+    let monitor = MultiMonitor::new(|s| println!("{}", s));
+
+    let cores = Cores::from_cmdline("0-3").unwrap();
```

Comment:
Why hardcode the cores?

Reply:
change to ` let cores = Cores::all()`

Reply:
Could be food to have a `--cores` parameter, but since this is only an example fuzzer ppl can change it if they want, of course.